scripts/test-node-cli-mode.sh

```diff
@@ -0,0 +1,17 @@
+# Prepack test input
+node ./bin/prepack.js ./test/node-cli/Simple.js --out Simple-test.js --compatibility node-cli
```

Comment:
It's checked into the repo already. https://github.com/facebook/prepack/tree/master/bin

It's an alias for `lib/prepack-cli.js` which gets there from the build.